docs/diginstroom/sip/1.0/profiles/newspaper.md

```diff
@@ -0,0 +1,346 @@
+It also allows to use the [MODS metadata schema](https://www.loc.gov/standards/mods/) for descriptive metadata, which is the default for describing newspaper content.
+
+**Permalink:** <https://data.hetarchief.be/id/sip/1.0/newspaper>
+
+**Example Directory structure:**
+
+```plaintext
+root_directory
+│──manifest-md5.txt
+│──bagit.txt
+│
+└──data
+    │──mets.xml
+    │──metadata
+    |   |──descriptive
+    |   |  └──dc.xml
+    |   |  └──mods.xml
+    |   └──preservation
+    |       └──premis.xml
+    │
+    └──representations
+        │──representation_1
+        │    │──mets.xml
+        │    │──data
+        │    |   |──file_1.tiff
+        │    │   └──...
+        │    │
+        │    └──metadata
+        │        └──preservation
+        │            └──premis.xml
+        │
+        │──representation_2
+        │    │──mets.xml
+        │    │──data
+        │    |   |──file_1.xml
+        │    │   └──...
+        │    │
+        │    └──metadata
+        │       └──preservation
+        │          └──premis.xml
+        │
+        └──representation_3
+            │──mets.xml
+            │──data
+            |  └── file_1.pdf
+            │
+            └──metadata
+               └──preservation
+                  └── premis.xml
+```
+
+## Requirements
+
+### General
+
+- A newspaper SIP MUST contain exactly one newspaper edition.
+- The newspaper edition MUST be digitised per page, i.e. each TIFF and/or ALTO XML file contained in their respective representation directories MUST represent exactly one page.
+  - <a id="pdf"></a>An exception to this requirement MAY be made with regards to the PDF file: if a PDF file is present, it SHOULD contain the contents of the entire newspaper edition (i.e. all pages are present in one single PDF file).
+- There MUST be exactly one IE present in the SIP, i.e. the newspaper edition.
+- Preservation metadata in the SIP MUST be limited to the PREMIS metadata schema.
+- There MUST be preservation metadata at the package level in the `preservation/premis.xml` file.
+- There MUST be preservation metadata at the representation level in the respective `preservation/premis.xml` files.
+- There MUST NOT be any descriptive metadata at the representation level.
+
+### Package METS
+
+- The `csip:CONTENTINFORMATIONTYPE` attribute MUST be set to `https://data.hetarchief.be/id/sip/1.0/newspaper`.
+
+### Package Descriptive Metadata
+
+- A descriptive metadata file `descriptive/mods.xml` MUST be present at the package level.
+- A descriptive metadata file `descriptive/dc.xml` MAY be present at the package level.
+- The `descriptive/mods.xml` file MUST follow the [MODS](https://www.loc.gov/standards/mods/v3/mods-3-7.xsd) metadata schema (v3.7.).
+- The `descriptive/dc.xml` file MUST follow the [DCTERMS](https://www.dublincore.org/schemas/xmls/qdc/dcterms.xsd) metadata schema.
```

Comment:
"if present". You can also merge this line and the one before that with the first two lines.

Reply:
Done.